http/pool.go

```diff
@@ -84,24 +87,31 @@ func (s *WSPool) OnConnected(c WSSpeaker) {
 	s.eventHandlersLock.RUnlock()
 }
 
-// OnDisconnected forwards the OnConnected event to event listeners of the pool.
-func (s *WSPool) OnDisconnected(c WSSpeaker) {
+func (s *WSPool) onDisconnected(c WSSpeaker) {
 	s.eventHandlersLock.RLock()
 	for _, h := range s.eventHandlers {
 		h.OnDisconnected(c)
 	}
 	s.eventHandlersLock.RUnlock()
 }
 
+// OnDisconnected forwards the OnConnected event to event listeners of the pool.
+func (s *WSPool) OnDisconnected(c WSSpeaker) {
+	logging.GetLogger().Debugf("OnDisconnected %s for pool %s ", c.GetHost(), s.GetName())
```

Comment:
I think you can put this line directly in the original function instead of creating a new one ? like what you did for wsIncomerPool

Reply:
@safchain - have eliminated `onDisconnected` now using `Ondisconnected`

Reply:
OnDisconnected()